example/hyperparam/search.py

```diff
@@ -0,0 +1,75 @@
+
+import os
+import re
+import sys
+import warnings
+
+import GPyOpt
+
+from subprocess import Popen, PIPE, STDOUT
+
+import numpy as np
+
+import mlflow
+import mlflow.sklearn
+import mlflow.tracking
+import mlflow.projects
+from mlflow.utils.logging_utils import eprint
+
+
+
+if __name__ == "__main__":
+    warnings.filterwarnings("ignore")
+    np.random.seed(40)
+    max_runs = int(sys.argv[1]) if len(sys.argv) > 1 else 50
+    max_p = int(sys.argv[2]) if len(sys.argv) > 2 else 4
+    alpha_min = float(sys.argv[3]) if len(sys.argv) > 3 else 0
+    alpha_max = float(sys.argv[4]) if len(sys.argv) > 4 else 1
+    l1_min = float(sys.argv[5]) if len(sys.argv) > 5 else 0
+    l1_max = float(sys.argv[6]) if len(sys.argv) > 6 else 1
+
+    bounds = [
+        {'name': 'alpha', 'type': 'continuous', 'domain': (alpha_min, alpha_max)},
+        {'name': 'l1_ratio', 'type': 'continuous', 'domain': (l1_min, l1_max)},
+    ]
+
+    def eval(parms):
+        print(parms)
+        print(parms.shape)
+        alpha, l1_ratio = parms[0]
+        cmd = ["mlflow", "run", os.path.abspath(os.path.dirname(__file__))]
+        cmd += ["-P", "alpha={}".format(alpha)]
+        cmd += ["-P", "l1_ratio={}".format(l1_ratio)]
+        print(" ".join(cmd))
+        proc = Popen(cmd, stdout=PIPE, stderr=STDOUT, env={
+            "PATH": os.environ.get("PATH"),
+            "LANG": os.environ.get("LANG")
+        },
```

Comment:
Yes, I had an issue with version of matplotlib (required by GPyOPt) that's why I started new process and parsed the runId from the output. Looks like using one from conda (instead of pip) resolved this. I'll update the PR.